controllers/topics/create.js

```diff
@@ -0,0 +1,37 @@
+const {Topics} = require('../../databases/models');
+const {insertTopics} = require('../../utils/post');
+
+module.exports = async (req, res) => {
+  try {
+    let {name} = req.body;
+    name = name.toLowerCase();
+    let topics = await Topics.findOne({
+      where: {
+        name,
+        deleted_at: null
+      }
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While querying the database, you are checking if a topic with the same name exists and is not deleted. This is good for ensuring uniqueness of active topics. However, it might be worth considering what should happen if a topic was previously deleted and now someone wants to create a new one with the same name. If this is allowed, then the current check would prevent that.



<!-- This is an auto-generated comment by OSS CodeRabbit -->